app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -10,8 +10,11 @@ def do_clone_task_check(clone_task_ref)
   end
 
   def find_destination_in_vmdb(vm_uid_hash)
-    ems_ref = vm_uid_hash.values.join("\\")
-    ManageIQ::Providers::Azure::CloudManager::Vm.find_by(:ems_ref => ems_ref.downcase)
+    # as azure vm name in base is case-sensitive, we don't want to downcase it
```

Comment:
Not sure the comment if truly necessary but if you want to keep it drop the "as".
`# Azure vm name is case-case-sensitive, we don't want to downcase it`